shared/django_apps/codecov_auth/models.py

```diff
@@ -167,6 +167,10 @@ def activated_user_count(self) -> int:
         # students and filter them out.
         return self.users.filter(owners__student=False).count()
 
+    @property
```

Comment:
Should we consider using [@cached_property](https://docs.python.org/3/library/functools.html#functools.cached_property) in some of these that hit the DB? Maybe the DB itself already has a cache that we can fallback on?

Reply:
we could - it's an interesting idea. I'm hoping that this won't be a heavy/expensive query, I'm going to add it as-is and if it's slow I'll upgrade it to `@cached_property`